.github/workflows/linux-workflow.yml

```diff
@@ -120,3 +136,20 @@ jobs:
       - name: Build PCSX2
         working-directory: ./build
         run: ../.github/workflows/scripts/linux/compile.sh
+
+      - name: Build AppImage Artifacts
+        env:
+          PLATFORM: ${{ matrix.platform }}
+        run: chmod +x ./.github/workflows/scripts/linux/appimage.sh && ./.github/workflows/scripts/linux/appimage.sh
```

Comment:
indentation is off, this would likely lead to some unexpected results.  I'd suggest committing the script with the executable flag as well, if you're on windows you can do that via:
```
git update-index --chmod=+x ./.github/workflows/scripts/linux/appimage.sh
```